src/pages/pupils/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug]/index.tsx

```diff
@@ -50,11 +50,29 @@ const PupilPageContent = ({
     updateCurrentSection(overrideSection);
   }
 
-  const { starterQuiz, exitQuiz } = curriculumData;
+  const {
+    starterQuiz,
+    exitQuiz,
+    lessonTitle,
+    subjectTitle,
+    subjectSlug,
+    yearTitle,
+    pupilLessonOutcome,
+  } = curriculumData;
 
   switch (currentSection) {
     case "overview":
-      return <PupilViewsLessonOverview />;
+      return (
+        <PupilViewsLessonOverview
+          lessonTitle={lessonTitle}
+          subjectTitle={subjectTitle}
+          subjectSlug={subjectSlug}
+          yearTitle={yearTitle ?? undefined}
+          pupilLessonOutcome={pupilLessonOutcome ?? undefined}
+          starterQuizNumQuestions={starterQuiz?.length ?? 0}
+          exitQuizNumQuestions={exitQuiz?.length ?? 0}
```

Comment:
I needed the number of questions in the quiz to display the count before the quiz has been started — so I couldn't get this value from the lesson engine state since this isn't populated until the first question is answered.

Reply:
yes that makes sense.